src/coreclr/crosscomponents.cmake

```diff
@@ -7,7 +7,7 @@ endif(CLR_CMAKE_HOST_ARCH_AMD64 AND (CLR_CMAKE_TARGET_ARCH_ARM OR CLR_CMAKE_TARG
 if (CLR_CMAKE_HOST_OS STREQUAL CLR_CMAKE_TARGET_OS)
     set (CLR_CROSS_COMPONENTS_LIST
         clrjit
-        jitinterface
+        jitinterface_${ARCH_HOST_NAME}
```

Comment:
Why do we need multiple jitinterface-wrapper libraries?

(Since crossgen2 basically ships selfcontained with the SDK, wouldn't the bitness of jitinterface.dll always match the bitness of coreclr.dll that the compiler is running with?)

Reply:
We don't need them in the shipping scenarios, but it enables some mix and matching of bits in some test infra work I'm lookin at. In particular, it makes it fairly simple to have a crossgen2 that is easy to use in crosstargeting scenarios in the core_root directory, especially on Windows. (Since the default available dotnet in our Windows build environment is 64bit only, this small tweak makes it easy to have a crossgen2 that will just work for developers)